src/taxonomy/export-modal/ExportModal.scss

```diff
@@ -0,0 +1,5 @@
+.taxonomy-export-modal {
+  .export-modal-radio {
+    width: 700px;
```

Comment:
Why this specific width?

Reply:
It's the size of the export modal. We need to expand the label to all the width of the modal so the user can select also on the white spaces. But in `Form.Radio` there is a parent `<div>` without any class, so I can't give a `width: 100%`.

![Captura desde 2023-12-12 14-10-58](https://github.com/openedx/frontend-app-course-authoring/assets/11827305/580ef13e-60c8-44db-9fe0-ef1e7b84632c)

Reply:
Can't you target the <div> anyways with a rule like `.export-modal .pgn__form-radio > div { width: 100%; }` ?

Reply:
It works, thanks. Updated here: https://github.com/openedx/frontend-app-course-authoring/pull/723/commits/98ea9de7eabd916da088a5a0a4a6964c99109e71